lib/pages/stock_exchange/components/stock_detail.dart

```diff
@@ -123,6 +124,14 @@ class _StockDetailState extends State<StockDetail> {
   }
 
   Widget _marketStatus(Stock company) {
+    final Stream<Int64> dayHighStream =
```

Comment:
```suggestion
    final dayHighStream =
```
No need to specify type ig

Reply:
Everything else is good to go 👍